scalableminds/webknossos
# Add JSON schema for zarr datasources

### URL of deployed dev instance (used for testing):
- https://___.webknossos.xyz

### Steps to test:
- edit the settings for a zarr dataset (e.g., add `https://data-humerus.webknossos.org/data/zarr/scalable_minds/l4dense_motta_et_al_demo/segmentation` via the new ZARR import front-end)
- editing the JSON should work without "invalid JSON" errors

### Issues:
- follow-up #6383 

------
